target/cheri-common/op_helper_cheri_common.c

```diff
@@ -416,7 +416,13 @@ void CHERI_HELPER_IMPL(cjalr(CPUArchState *env, uint32_t cd,
 
     const cap_register_t *cbp = get_readonly_capreg(env, cb);
     const target_ulong cursor = cap_get_cursor(cbp);
+#ifdef TARGET_RISCV
```

Comment:
I'm not particularly keen on the ifdef TARGET_RISCV in cheri-common, but we can't really push it down into translate.c since we're doing the addition in the helper.